bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Accumulation/AccumulateShares.cs

```diff
@@ -0,0 +1,196 @@
+using ElectionGuard.Ballot;
+using ElectionGuard.Decryption.Accumulation;
+using ElectionGuard.Decryption.Decryption;
+using ElectionGuard.Decryption.Shares;
+using ElectionGuard.Decryption.Tally;
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.Accumulation;
+
+public static class AccumulateSharesExtensions
+{
+    public static AccumulatedTally AccumulateShares(
```

Comment:
Accumulation is now separated out from decryption so that the accumulated values can be used to create the challenges.